package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Zendesk, Inc.",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=4"
+    "node": ">=12"
```

Comment:
We should bump this as a Major and make sure we add our changes into changelog.md

Otherwise LGTM

Reply:
👍 Yes, this should be >=10 instead too.